include/ccf/endpoint_registry.h

```diff
@@ -39,6 +39,25 @@ namespace ccf::endpoints
     PathTemplateSpec spec;
   };
 
+  struct RequestCompletedEvent
+  {
+    std::string method = "";
```

Comment:
verb?

Reply:
I think this is `method` in HTTP terminology. We only use `verb` in some places for historical reasons (converting from JSON-RPC, which have a _method_ which is more like a HTTP _path_), but this seems like the sane name in a new API?

Reply:
My bad, I thought there were verbs, and the word is used, but you are right that they formally are methods.